src/server/scripts/Spells/spell_shaman.cpp

```diff
@@ -1436,6 +1438,45 @@ class spell_sha_windfury : public SpellScriptLoader
     }
 };
 
+// 192078 - Windrush Totem (Spell)
+//  12676 - AreaTriggerId
+struct areatrigger_sha_windrush_totem : AreaTriggerAI
+{
+    static uint32 const REFRESH_TIME = 4500;
```

Comment:
should this be constexpr ?

Reply:
sounds good, kinda unused to constexprs yet

Reply:
It seems it has to be static constexpr according to CI logs